src/Http/Http.Extensions/test/ParameterBindingMethodCacheTests.cs

```diff
@@ -1188,6 +1242,28 @@ public static void BindAsync(HttpContext context)
             => throw new NotImplementedException();
     }
 
+    private class BindAsyncFromImplicitStaticAbstractInterface : IBindableFromHttpContext<BindAsyncFromImplicitStaticAbstractInterface>
```

Comment:
Add a test that has both `IBindableFromHttpContext` and an implemented BindAsync method (without Parameter) to see if it chooses the `IBindableFromHttpContext`.